src/img/senvcfgreg.edn

```diff
@@ -6,20 +6,18 @@
 (def left-margin 30)
 (def right-margin 30)
 (def boxes-per-row 32)
-(draw-column-headers {:height 20 :font-size 18 :labels (reverse ["0" "" "1" "3" "4" "5" "6" "" "7" "" "8" "" "" "" "" "" "" "" "" "" "" "" "" "27" "28" "" "29" "" "30" "" "" "SXLEN-1"])})
+(draw-column-headers {:height 20 :font-size 18 :labels (reverse ["0" "" "1" "3" "4" "5" "6" "" "7" "" "8" "" "" "" "" "" "" "" "" "" "" "" "" "" "27" "28" "" "" "" "" "" "SXLEN-1"])})
 
-(draw-box (text "WPRI" {:font-weight "bold" :font-size 20}) {:span 4})
+(draw-box (text "WPRI" {:font-weight "bold" :font-size 20}) {:span 6})
```

Comment:
This has the same problem as menvcfg

<img width="582" alt="image" src="https://github.com/riscv/riscv-cheri/assets/59061376/8a2c99fd-deb2-4f4d-a494-f98fb14640ab">